ee/codegen_integration/conftest.py

```diff
@@ -34,10 +36,29 @@ def _get_fixture_paths(root: str) -> Tuple[str, str]:
             "simple_inline_subworkflow_node",
             "simple_guardrail_node",
             "simple_map_node",
-            "simple_code_execution_node",
         }
     )
 )
 def code_to_display_fixture_paths(request) -> Tuple[str, str]:
     root = request.param
     return _get_fixture_paths(root)
+
+# Save the original `open`
+builtin_open = open
```

Comment:
It doesn't look like this is actually used?

Reply:
It's used a few lines below